com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDCamera.cs

```diff
@@ -1125,6 +1129,13 @@ unsafe internal void UpdateShaderVariablesGlobalCB(ref ShaderVariablesGlobal cb,
             float ct = time;
             float pt = lastTime;
 #if UNITY_EDITOR
+            // Apply editor mode time override if any.
+            if (animateMaterials)
```

Comment:
Important addition to HDCamera to achieve deterministic motion vector results for vertex animation.